### PR TITLE
Add WNP 110 for EU (Fixes #12664)

### LINKED ISSUE #12664: WNP 110 (EU)
## Description

**Needs to be live before 2023-02-14**

Requirements:

- DE:
  - 50% of users of all users get V1 (PoD)
  - 50% of users of all users get V2 (save GIF)
- UK & FR:
  - 100% of users get V3 (save GIF)

Assets:

- Figma: https://www.figma.com/file/8fVLbzzAd40lSSLuOwCfnl/EU_Firefox_WNP_21-23?node-id=2406%3A1281&t=wTm7Kvv3gb4Gmtrr-1
- Copy: https://docs.google.com/document/d/1N2zYCNVbRjYAyppS6ZTjsi20sJCLTTFvAxKHjRR24so/edit#heading=h.qfrh7y1abpje
- Drive: https://drive.google.com/drive/folders/1olMwlTjDU9GiPuofPKMBXJKBSar2pafm?usp=share_link

Additional requirement:

- Fire a GA event for all variations that will log if someone has recently signed in to Pocket via FxA or not. We've [done that here](https://github.com/mozilla/bedrock/blob/main/media/js/firefox/whatsnew/whatsnew-108.es6.js) previously before.

---

#### Success Criteria
- [ ] Pages need to be live before release date on 2023-02-14

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Adds WNPs for Firefox 110 for DE, FR, GB

## Issue / Bugzilla link

https://github.com/mozilla/bedrock/issues/12664

## Testing

- http://localhost:8000/de/firefox/110.0/whatsnew/?v=1
- http://localhost:8000/de/firefox/110.0/whatsnew/?v=2
- http://localhost:8000/fr/firefox/110.0/whatsnew/
- http://localhost:8000/en-GB/firefox/110.0/whatsnew/
- http://localhost:8000/en-US/firefox/110.0/whatsnew/?geo=gb

### CODE REVIEW

**Comment by alexgibson** [comment]
Note to self:

Still TODO: "Fire a GA event for all variations that will log if someone has recently signed in to Pocket via FxA or not. "

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx110-de-v1.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  {% call split(
+    block_class='mzp-l-split-reversed mzp-l-split-center-on-sm-md mzp-t-content-lg wnp-content-main',
+    body_class='mzp-l-split-v-start',
+    media_class='mzp-l-split-v-start',
+    media_after=True,
+    image=picture(
+      url="img/firefox/whatsnew/whatsnew110-eu/pocket-de.gif",
+      sources=[
+          {
+              "media": "(prefers-reduced-motion: reduce)",
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx110-de-v1.html:39]: 🔥

File: bedrock/firefox/tests/test_base.py [hunk 2]
```diff
@@ -787,6 +787,64 @@ def test_fx_109_0_0_feature_variation_en(self, render_mock):
...
+        req = self.rf.get("/firefox/whatsnew/?v=2")
+        req.locale = "de"
+        self.view(req, version="110.0")
+        template = render_mock.call_args[0][1]
+        assert template == ["firefox/whatsnew/whatsnew-fx110-de-v2.html"]
+
+    @override_settings(DEV=True)
+    def test_fx_110_0_0_fr(self, render_mock):
+        """Should use whatsnew-fx110-fr template for fr locale"""
+        req = self.rf.get("/firefox/whatsnew/?v=2")
```
Comment (craigcook) [inline bedrock/firefox/tests/test_base.py:822]: Probably don't need the `?v=2` in the URL here, I suspect that was copypasta from the test above.

**Review by craigcook — APPROVED** [review]
Looks great, but one question: should we do a proper dark mode for DE v2? It feels odd for the header and footer to switch to dark mode but not the body. Changing text and background colors is easy enough but we'd also need an inverted version of the wordmark image (I can make that if you need it). Alternatively we could remove the dark mode from v2 but that would mean some refactoring of the CSS to keep it for v1. Up to you if you want to put any effort into it or leave it as-is, I just wanted to bring it up.

r+wc 🥜

**Comment by alexgibson** [comment]
> Looks great, but one question: should we do a proper dark mode for DE v2? It feels odd for the header and footer to switch to dark mode but not the body. Changing text and background colors is easy enough but we'd also need an inverted version of the wordmark image (I can make that if you need it). Alternatively we could remove the dark mode from v2 but that would mean some refactoring of the CSS to keep it for v1. Up to you if you want to put any effort into it or leave it as-is, I just wanted to bring it up.

For V2 this is the single design provided by the EU studio team for both light and dark mode. Rather than revisit design at this stage after things have already been approved by everyone, I'm inclined just to leave it as-is.

**Comment by alexgibson** [comment]
Thanks @craigcook !

**Pull request merged**